appserver/security/webservices.security/src/main/java/com/sun/enterprise/security/webservices/CommonServerSecurityPipe.java

```diff
@@ -46,6 +47,7 @@
 /**
  * This pipe is used to do Jakarta Authentication security
  */
+@SuppressWarnings("deprecation")
```

Comment:
Isn't better to see all deprecations?

Reply:
Normally yes, but if I'm not mistaken the Pipe was deprecated in favour of the Tube very shortly after the Pipe was introduced (relatively speaking), i.e. somewhere in the golden days of XML around 2005 or so.

But the Pipe has been in use more than the Tube ever since, which is close to 20 years now. It's not something that's there and should go away since it's barely used still. The Pipe is pretty much the thing you always use. It's a bit of a mess really.

I already asked to undeprecate the Pipe, since after 20 years I think we can conclude deprecating it was perhaps done in error?

cc @lukasj